### ISSUE TITLE
Ограничить совмместимость с Nodejs 18.13.0 LTS

### ISSUE THREAD

**Issue by kernusr** [crypto-pro-web/crypto-pro-js#46]

**Comment by khidirbekov** [comment]
Привет! Использовал ваш пакет, вместо crypto-pro, т.к. у вас реализованы методы со-подписи, а с этим вопрос в том репозитории застопился и год уже нет как подвижек.

В этом пакете столкнулся с тем, что библиотека работает только с 16 версией ноды. Можете подсказать, пожалуйста, из-за чего в `package.json` был добавлен параметр `"engines": { "node": "~16" }`? Можно ли будет там указать в качестве параметра ">= 16", чтобы была поддержка ноды по-новее?

**Comment by kernusr** [comment]
> Можно ли будет там указать в качестве параметра ">= 16", чтобы была поддержка ноды по-новее?

Вы попробовали запустить сборку на более новой ноде? Какой-то результат получили?

В ближайшее время планирую обновить проект, для совместимости с актуальными lts версиями node js и webpack

**Comment by khidirbekov** [comment]
Да, пробовал, для это в параметры билда нужно прокинуть `NODE_OPTIONS=--openssl-legacy-provider`. Но есть и минус, ломается сборка на 16 ноде, нужна версия по новее

Репа, где я пробовал это сделать https://github.com/khidirbekov/crypto-pro-js/tree/master
